install_config/aggregate_logging.adoc

```diff
@@ -300,6 +300,54 @@ when `openshift_logging_use_ops` is set to `true`.
 |Equivalent to `openshift_logging_kibana_replica_count` for Ops cluster
 when `openshift_logging_use_ops` is set to `true`.
 
+|`openshift_logging_es_allow_external`
+|Set to `true` to expose Elasticsearch as a reencrypt route. Set to `false` by
+default.
+
+|`openshift_logging_es_hostname`
+|The external facing hostname to use for the route and the TLS server
+certificate. The default is set to `es`.
+
+`openshift_master_default_subdomain`,
+for example, if `openshift_master_default_subdomain=example.test`, then the
+default value of `openshift_logging_es_hostname` will be `es.example.test`.
```

Comment:
I think I confused the above for another example at first. Perhaps the second para above can be:

"For example, if `openshift_master_default_subdomain` is set to `=example.test`, then the default value of `openshift_logging_es_hostname` will be `es.example.test`."

Reply:
That is much better, thank you. I took a bit with that one, moved on, and never made it back.